CONTRIBUTING.md

```diff
@@ -38,10 +38,13 @@
 -----
 
 ### Setup development environment
-From the top-level directory:
+The TERRA_CLI_DOCKER_MODE environment variable controls Docker support. Set it to
+   DOCKER_NOT_AVAILABLE (default) to skip pulling the Docker image
+   or DOCKER_AVAILABLE to pull the image (requires Docker to be installed and running).
+Then, from the top-level directory, run:
 ```
 source tools/local-dev.sh
-terra
+build/install/terra-cli/bin/terra
```

Comment:
I don't follow why you need this change, because `local-dev.sh` aliases `terra` for you.

Reply:
Good point. I was working with different install modes on this PR and another one, and it's easy to get confused.